### ISSUE TITLE
Temporary stall 

### ISSUE THREAD

**Issue by dahankzter** [scylladb/scylla#1403]
*Installation details*
Scylla version (or git commit hash): 1.1
Cluster size: 3
OS (RHEL/CentOS/Ubuntu/AWS AMI): RHEL 7

I just noticed noticed when an app dynamically tried to create a keyspace.
I checked my query but it was no different than what used to work.
I then proceeded to try via the cqlsh which connected very fast.
The query however timed out.
I looked in the log and saw a lot of the lines below. Other than that nothing really weird.
Decided to restart scylla on all nodes and after quite a long time, maybe 1 minute the nodes was restarted.
After this the queries are working just fine and super fast.

Any ideas? Can I check something? I have it on the todo to upgrade but a little bit too busy atm.

```bash
Jun 28 15:29:24 app66 scylla[21019]:  [shard 0] rpc - client x.y.z.z: unknown verb exception 6 ignored
Jun 28 15:29:25 app66 scylla[21019]:  [shard 0] rpc - client x.y.z.z: unknown verb exception 6 ignored
Jun 28 15:29:25 app66 scylla[21019]:  [shard 0] rpc - client x.y.z.z: unknown verb exception 6 ignored
Jun 28 15:29:26 app66 scylla[21019]:  [shard 0] rpc - client x.y.z.z: unknown verb exception 6 ignored

```

**Comment by gleb-cloudius** [comment]
On Tue, Jun 28, 2016 at 06:36:04AM -0700, Henrik Johansson wrote:
> *Installation details*
> Scylla version (or git commit hash): 1.1
> Cluster size: 3
> OS (RHEL/CentOS/Ubuntu/AWS AMI): RHEL 7
> 
> I just noticed noticed when an app dynamically tried to create a keyspace.
> I checked my query but it was no different than what used to work.
> I then proceeded to try via the cqlsh which connected very fast.
> The query however timed out.
> I looked in the log and saw a lot of the lines below. Other than that nothing really weird.
> Decided to restart scylla on all nodes and after quite a long time, maybe 1 minute the nodes was restarted.
> After this the queries are working just fine and super fast.
> 
> Any ideas? Can I check something? I have it on the todo to upgrade but a little bit too busy atm.
> 
> ```bash
> Jun 28 15:29:24 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.164: unknown verb exception 6 ignored
> Jun 28 15:29:25 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> Jun 28 15:29:25 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> Jun 28 15:29:26 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> Jun 28 15:29:26 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> Jun 28 15:29:27 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> Jun 28 15:29:27 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.164: unknown verb exception 6 ignored
> Jun 28 15:29:28 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> Jun 28 15:29:28 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.164: unknown verb exception 6 ignored
> Jun 28 15:29:29 app66.prod.content.eniro scylla[21019]:  [shard 0] rpc - client 172.27.164.147: unknown verb exception 6 ignored
> ```
> 
Those messages by themselves are benign if they are printed during node
bootup (recent versions should not print then any longer). If you saw
them been printed while you created namespace it means that the node was
stuck during bootup for some reason.

--
			Gleb.

**Comment by dahankzter** [comment]
It was probably logged during bootup. I did only attempt restart because the cluster would not accept the keyspace creation however.

**Comment by tzach** [comment]
@dahankzter @gleb-cloudius any update on this issue?
Can it be reproduced?

**Comment by slivne** [comment]
@dahankzter is there additional info to reproduce ?

**Comment by dahankzter** [comment]
Not really. I haven't seen it since either. Don't like temp issues but perhaps it was fixed since upgrading to 1.2.
